### PR TITLE
Clarifying device support

### PR DESCRIPTION
**Pull request by jovirkku**
Let's clarify the supported devices and the availability of the Sailfish licence.

### CODE REVIEW

**Comment by jovirkku** [comment]
" AppSupport[^3]  version/<br />API level[^4]" replaced with  "AppSupport[^3] /<br />API level[^4]".

File: Support/Supported_Devices/README.md [hunk 1]
```diff
@@ -5,15 +5,20 @@ layout: default
...
-| Xperia 10 V SIM+uSD+eSIM   | XQ-DC54             | Android 13<br />                    | 5.4 / 13         | NA/NA                                       | 4.6.0<br />Sauna<br />2024       |
-| Xperia 10 IV SIM+uSD+eSIM  | XQ-CC54             | Android 13<br />                    | 5.4 / 13         | -"-                                         | -"-                              |
+| Device                     | Model               | Required stock Android version[^1]  | Kernel / BSP[^2] | AppSupport[^3] /<br />API level[^4] | First Sailfish OS release        |
+| -------------------------- | ------------------- | :---------------------------------: | :--------------: | ------------------------------------------- | -------------------------------- |
+| Xperia 10 V SIM+uSD+eSIM   | XQ-DC54             | Android 14<br />                    | 5.4 / 14         | 11 / 30                                     | 4.6.0<br />Sauna<br />2024       |
+| Xperia 10 IV SIM+uSD+eSIM  | XQ-CC54             | Android 14<br />                    | 5.4 / 14         | 11 / 30                                     | 4.6.0<br />Sauna<br />2024       |
```
Comment (mkosola) [inline Support/Supported_Devices/README.md:21]: I think we need to keep Android 13 here as Emma is offering currently only Android 13 SW (65.1.A.5.65)

Comment (jovirkku) [reply Support/Supported_Devices/README.md:21]: For an Xperia 10 V that was updated to Android 14 (Build ID 68.1.A.2.230), Emma has only the Android 13 build version 68.0.A.0.797 available. On the other hand, according to the Forum, many people have already updated their phones to Android 14.  Hopefully, Sony will add a new Android 14 build to Emma soon. Could we rely on this? I think our goal has been Android 14.

Comment (mkosola) [reply Support/Supported_Devices/README.md:21]: I think "Required stock" Android version should be 13 and and if somebody has updated to 14 they can use it.

Comment (jovirkku) [reply Support/Supported_Devices/README.md:21]: Yes, perhaps that is the best way.

Comment (jovirkku) [reply Support/Supported_Devices/README.md]: "Android 13 (14 is ok)"

**Review by mkosola — APPROVED** [review]

**Comment by jovirkku** [comment]
Raine, is this ok now? I have nothing to add at this point.

**Review by rainemak — APPROVED** [review]

**Pull request merged**